src/frontend/Sider/Behandling/VedtakOgBeregning/Læremidler/InnvilgeVedtak/InnvilgeLæremidler.tsx

```diff
@@ -55,15 +59,29 @@ export const InnvilgeLæremidler: React.FC<{
         }
     };
 
+    const validerForm = (): boolean => {
+        const vedtaksperiodeFeil = validerVedtaksperioder(vedtaksperioder);
+        settVedtaksperiodeFeil(vedtaksperiodeFeil);
+
+        return vedtaksperiodeFeil.every(
+            (vedtaksperiode) => vedtaksperiode === undefined || isValid(vedtaksperiode)
+        );
```

Comment:
Hvorfor trengs `vedtaksperiode === undefined`?
Tror man kanskje kan gjøre noe slikt
```suggestion
        return isValid(vedtaksperiodeFeil)
```

Reply:
Det var bedre!

Vi må sørge for at antall elementer i `vedtaksperiodeFeil`-listen samsvarer med antall vedtaksperioder i UI-et, siden vi bruker indeksene for å knytte riktig feilmelding til riktig element. Jeg trodde at elementene i listen ble satt til `undefined` hvis det ikke var noen feil, og at isValid ikke kunne brukes på en liste. Ser jeg at det ikke stemmer, så jeg oppdaterer koden!